portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/calendar/GoogleCalendarToModelConverter.java

```diff
@@ -0,0 +1,62 @@
+package org.dataportabilityproject.serviceProviders.google.calendar;
+
+import com.google.api.services.calendar.model.CalendarListEntry;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
+import com.google.api.services.calendar.model.EventDateTime;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.dataportabilityproject.dataModels.calendar.CalendarAttendeeModel;
+import org.dataportabilityproject.dataModels.calendar.CalendarEventModel;
+import org.dataportabilityproject.dataModels.calendar.CalendarModel;
+
+public class GoogleCalendarToModelConverter {
```

Comment:
Not sure if it would make sense for these converters, but you might be able to use Converter<A, B> with doForward and doBackward for a more consistent API. See LegacyPortabilityJobConverter

Reply:
I've been following the model Jim outlined earlier, but always happy to revisit!

Reply:
ok SG!